toolkits/landcover/test/helpers/test-image.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Returns a single constant ee.Image comprised of the bands and values in the
+ * specified dictionary.
+ *
+ * @param {*} values A dictionary containing band names as keys and
+ *    corresponding numeric values. The values are used to build an ee.Image
+ *    with a single constant value in each respective band.
+ * @returns {!ee.Image}
+ */
+var collectBands = function(values) {
+  var result;
+  // Iterate over values dictionary to build Image, converting each value to
+  // the appropriate EE type.
+  for (var band in values) {
+    var value = values[band];
+    // If/when other types are needed they can be added here.
+    var img = ee.Image(value).rename(band).int16();
+    if (result) {
+      result = result.addBands(img);
+    } else {
+      result = img;
+    }
+  }
+  return result;
+};
+
+/**
+ * Returns a single constant ee.Image comprised of the bands and values in the
+ * specified dictionary.
+ *
+ * @param {*} values A dictionary containing band names as keys and
+ *    corresponding numeric values. The values are used to build an ee.Image
+ *    with a single constant value in each respective band.
+ * @param {string} opt_date An optional start date for the image, in YYYY-MM-DD
+ *    format.
+ * @returns {!ee.Image}
+ */
+var create = function(values, opt_date) {
```

Comment:
If the date is optional, then there's no reason to have two calls.
This should be merged with the call above.

Also, the opt_prefix is no longer recommended.